XamlStyler.Console/Options.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System.Collections.Generic;
```

Comment:
Can you add our standard header to the top of all code files?

"// © Xavalon. All rights reserved."

Reply:
Done